main.rb

```diff
@@ -0,0 +1,76 @@
+require_relative './app'
+
+def run_option
+  puts 'Welcome to the School Library App!'
```

Comment:
- [ ] @MuhammadUsmanAslam Great work on your interface, however, I would recommend you ensure the `Welcome to the Schoo Library App!` only appears when the app is launched and not whenever a person choose the options in the menu as shown below 
![image](https://github.com/MuhammadUsmanAslam/OOP-school-library/assets/61408860/53a63ea7-deb0-4548-882b-b428fa67e010)